src/main/java/edu/unc/lib/boxc/migration/cdm/status/DestinationsStatusService.java

```diff
@@ -138,7 +138,7 @@ public void reportDestinationStats(int totalObjects, Verbosity verbosity) {
                 showFieldListValues(newColls);
             }
         } catch (IOException e) {
-            outputLogger.info("Failed to load destinations mapping: {}", e.getMessage());
+            outputLogger.info("Failed to load destinations mapping: {}", e);
```

Comment:
Might make sense to add in a regular logger line here and just send the message to the outputLogger